### PR TITLE
Update install doc to match release v0.19.8

### LINKED ISSUE #150: Getting error when trying to enable coscheduler following develop instructions and tutorial
Moved from https://github.com/kubernetes/website/issues/21128,

Using minikube, I compiled the scheduler-plugins image (according to [these instructions](https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/doc/develop.md)), and set the image in my `/etc/kubernetes/manifests/kube-scheduler.yaml` file of my master node. I.e:
```
apiVersion: v1
kind: Pod
metadata:
  creationTimestamp: null
  labels:
    component: kube-scheduler
    tier: control-plane
  name: kube-scheduler
  namespace: kube-system
spec:
  containers:
  - command:
    - kube-scheduler
    - --authentication-kubeconfig=/etc/kubernetes/scheduler.conf
    - --authorization-kubeconfig=/etc/kubernetes/scheduler.conf
    - --config=/etc/kubernetes/coscheduling-config.yaml
    - --bind-address=127.0.0.1
    - --kubeconfig=/etc/kubernetes/scheduler.conf
    - --leader-elect=false
    - --port=0
    image: localhost:5000/scheduler-plugins/kube-scheduler:latest   <---- The compiled custom scheduler
    imagePullPolicy: IfNotPresent
    livenessProbe:
      failureThreshold: 8
 .....
```
Which I thought would create the scheduler pod with the custom scheduler image but I get the error message:
```
W0219 17:10:50.073294       1 client_config.go:613] error creating inClusterConfig, falling back to default config: open /var/run/secrets/kubernetes.io/serviceaccount/token: no such file or directory
couldn't create scheduler using provider "DefaultProvider": initializing profiles: creating profile for scheduler name default-scheduler: error initializing plugin "Coscheduling": failed to init rest.Config: invalid configuration: no configuration has been provided, try setting KUBERNETES_MASTER environment variable
```

@Huang-Wei

### PR DESCRIPTION
**Pull request by Huang-Wei**
Part of #161.

Updated the installation doc to match release v0.19.8.

Fixes #150.

### CODE REVIEW

**Comment by Huang-Wei** [comment]
cc @seanmalloy @xq2005 

/assign @cwdsuzhou @denkensk

File: doc/install.md
Comment (Huang-Wei) [inline doc/install.md:248]: @cwdsuzhou @denkensk could you help investigate why the status is not displayed? Better to compare the RBAC/CRD with your production settings.

File: manifests/install/all-in-one.yaml
Comment (Huang-Wei) [inline manifests/install/all-in-one.yaml:1]: @cwdsuzhou @denkensk could you help compare the RBAC/CRD with your production settings? to ensure the correctness and the privileges are not over-exposed. Thanks in advance.

File: manifests/install/all-in-one.yaml
Comment (Huang-Wei) [inline manifests/install/all-in-one.yaml:10]: I missed `patch` here; otherwise scheduler reports error:

```
E0317 01:20:22.700494       1 core.go:203] podgroups.scheduling.sigs.k8s.io "pg1" is forbidden: User "system:kube-scheduler" cannot patch resource "podgroups" in API group "schedul
ing.sigs.k8s.io" in the namespace "default"
```

File: doc/install.md
Comment (Huang-Wei) [reply doc/install.md:248]: After appending the 2nd commit, a PodGroup with matching Pods can display its `status`, but still empty for a PodGroup without any matching Pods.

File: manifests/install/all-in-one.yaml
Comment (cwdsuzhou) [reply manifests/install/all-in-one.yaml:10]: yep, we miss patch here

File: manifests/install/all-in-one.yaml
Comment (cwdsuzhou) [inline manifests/install/all-in-one.yaml:47]: we have not enabled `status` subresource of podgroup, so `podgroups/status` is not necessary

Comment (Huang-Wei) [reply manifests/install/all-in-one.yaml:47]: thanks. Could you help verify that when a PodGroup is created (without any matching pods), is its status displayed properly?

Comment (cwdsuzhou) [reply manifests/install/all-in-one.yaml]: > And if I create a deployment with 2 pods, the `phase` gets changed to `PreScheduling`, but if I delete the deployment, its phase stays in `PreScheduling` (I'd expect it to be updated back to `Pending`)

This may be a bug the status would not be back to `Pending`

File: manifests/install/all-in-one.yaml
Comment (cwdsuzhou) [reply manifests/install/all-in-one.yaml:1]: Hi, coscheduling works ok in my cluster using the the RBAC(changed in this PR) and CRD

image
  

Since pod group controller is not necessary now, we did not deploy in our production environment.

**Review by cwdsuzhou — APPROVED** [review]

File: manifests/install/all-in-one.yaml
Comment (Huang-Wei) [reply manifests/install/all-in-one.yaml]: > This may be a bug the status would not be back to Pending

I will create an issue tracking this.

BTW: could you help confirm this issue:

> after I updated controller SA's rbac setting (replace Update with Patch), it shows properly. But it's not showing the number:

Comment (cwdsuzhou) [reply manifests/install/all-in-one.yaml]: In my test environment, as I showed https://github.com/kubernetes-sigs/scheduler-plugins/pull/163#discussion_r595711417,  we can see number.

The controller is deployed based on the manifests in this PR

File: doc/install.md
Comment (cwdsuzhou) [inline doc/install.md:38]: seem the sequence is wrong. It is always `1.`

Comment (Huang-Wei) [reply doc/install.md:38]: It's intentional, it will be rendered properly.

File: manifests/install/all-in-one.yaml
Comment (Huang-Wei) [reply manifests/install/all-in-one.yaml]: Created #166 to track the UX issues.

**Comment by Huang-Wei** [comment]
Commits squashed and some UX issues are tracked in #166. Please take another look.

/hold cancel

**Comment by Huang-Wei** [comment]
@cwdsuzhou would you minding re-lgtm?

File: doc/install.md
Comment (seanmalloy) [inline doc/install.md:257]:
```suggestion
[install doc](https://github.com/kubernetes-sigs/scheduler-plugins/blob/release-1.18/doc/install.md) in
branch `release-1.18`.
```

Comment (Huang-Wei) [reply doc/install.md]: Thanks for the catch. I put this paragraph into a section `Instal lower-version releases`.

**Comment by seanmalloy** [comment]
/lgtm
/hold

Adding a hold so that @cwdsuzhou can review one more time before merge. Please feel free to remove the hold once you are ready for this to merge. Thanks!

**Comment by Huang-Wei** [comment]
Kindly ping @@cwdsuzhou for taking another look.

File: doc/install.md
Comment (cwdsuzhou) [inline doc/install.md:173]: always get `1.`, seems not intentional

**Comment by cwdsuzhou** [comment]
minor hits.  Otherwise, lgtm

File: doc/install.md
Comment (Huang-Wei) [reply doc/install.md:173]: When the list is long and frequently modified, it's not easy to always make its numbers ordered accurately :) So keep all as `1.` and make them indented well, markdown will do the rest things.

**Pull request merged**
